IFC4x3/Properties/m/MitigatedRiskLikelihood_2mh3cZyunB0ONymM2LqXn/DocProperty.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="MitigatedRiskLikelihood_2mh3cZyunB0ONymM2LqXn" Name="MitigatedRiskLikelihood" UniqueId="b0ac39a3-f38c-4b01-85fc-c16095d21c7f" PropertyType="p_enumeratedvalue" PrimaryDataType="IfcLabel">
```

Comment:
Not sure why I have this one twice. Seems to be the same GUID. Think its file renamed.

Reply:
fixed